dev/io.openliberty.concurrent.internal/resources/io/openliberty/concurrent/internal/resources/CWWKCMessages.nlsprops

```diff
@@ -45,3 +45,7 @@ CWWKC1203.duplicate.context.useraction=Update the application, libraries, or bot
 CWWKC1204.not.serializable=CWWKC1204E: Cannot create a serializable contextual proxy that propagates thread context {0} which is not serializable.
 CWWKC1204.not.serializable.explanation=Third-party thread context types are not compatible with serialization and cannot be configured to be propagated for serializable contextual proxies.
 CWWKC1204.not.serializable.useraction=Update the application to request a non-serializable contextual proxy or ensure that no third-party context types are configured to be propagated.
+
+CWWKC1205.qualifier.class.not.found=CWWKC1205E: Qualifier annotation {0}, which is defined by the application deployment descriptor, could not be loaded by application classloader.
```

Comment:
Would it be OK to instead say this?
The {0} qualifier annotation
IBM Style is to place the variable before the noun object. If the variable is such that it makes the sentence difficult to read or understand, we can leave it as is.

Also, recommend changing
...could not be loaded by application classloader.
to
...could not be loaded by the application classloader.